nnvm/include/nnvm/top/nn.h

```diff
@@ -313,6 +313,64 @@ struct LayoutTransformParam : public dmlc::Parameter<LayoutTransformParam> {
   }
 };
 
+struct MultiBoxPriorParam : public dmlc::Parameter<MultiBoxPriorParam> {
+  Tuple<float> sizes;
+  Tuple<float> ratios;
+  Tuple<float> steps;
+  Tuple<float> offsets;
+  bool clip;
+
+  DMLC_DECLARE_PARAMETER(MultiBoxPriorParam) {
+    DMLC_DECLARE_FIELD(sizes).set_default(Tuple<float>({1.0}))
+      .describe("List of sizes of generated MultiBoxPriores.");
+    DMLC_DECLARE_FIELD(ratios).set_default(Tuple<float>({1.0}))
+    .describe("List of aspect ratios of generated MultiBoxPriores.");
+    DMLC_DECLARE_FIELD(steps).set_default(Tuple<float>({-1.0, -1.0}))
+    .describe("Priorbox step across y and x, -1 for auto calculation.");
+    DMLC_DECLARE_FIELD(offsets).set_default(Tuple<float>({0.5, 0.5}))
+    .describe("Priorbox center offsets, y and x respectively.");
+    DMLC_DECLARE_FIELD(clip).set_default(false)
+    .describe("Whether to clip out-of-boundary boxes.");
+  }
+};
+
+struct MultiBoxDetectionParam : public dmlc::Parameter<MultiBoxDetectionParam> {
+  bool clip;
+  float threshold;
+  float nms_threshold;
+  bool force_suppress;
+  int nms_topk;
+  Tuple<float> variances;
+  DMLC_DECLARE_PARAMETER(MultiBoxDetectionParam) {
+    DMLC_DECLARE_FIELD(clip).set_default(true)
+      .describe("Clip out-of-boundary boxes.");
+    DMLC_DECLARE_FIELD(threshold).set_default(0.01)
+    .describe("Threshold to be a positive prediction.");
+    DMLC_DECLARE_FIELD(nms_threshold).set_default(0.5)
```

Comment:
is NMS stripped out of this op or not? I saw a standalone NMS operator below.

Reply:
Yes. NMS is now a standalone operator. multi box_detection is still preserved so that MulriboxDetection in MXNet can be directly converted.

Reply:
The conversion process does not have to be one to one, so we can break it into sequence, maybe  @zhreshold can have specific thouoghts on what that sequence is.